app/services/manage_braintree_donation.rb

```diff
@@ -0,0 +1,58 @@
+class ManageBraintreeDonation
```

Comment:
I feel like @osahyoun's style of namespacing classes with modules would work well for these classes that serve the purpose of sending actions to (or generally, coordinating with) ActionKit. That way we can have them all namespaced the same way and located in the same directory. E.g. `ActionkitHandler :: ManageBraintreeDonation` and `ActionkitHandler :: ManageAction`.